app/views/application/_email-confirmation-reminder.html.erb

```diff
@@ -1,6 +1,8 @@
-<section class="email-reminder" aria-label="Notice" role="region">
-  <p class="govuk-body govuk-!-margin-0">
-    <%= sanitize(t("confirm.intro.#{confirmation_banner_prompt_type}")) %>
-    <a href="<%= new_user_confirmation_path %>" class="govuk-link"><%= t("confirm.link_text") %></a>
-  </p>
-</section>
+<% if show_confirmation_reminder? %>
+  <section class="user-reminder" aria-label="Notice" role="region">
```

Comment:
I feel like this should be a component but I can't see one that we have already that matches this design - plus you've not written this frontend, it was already here, so probably out of scope for this PR anyway.